### PR TITLE
Datasource config was not mapped to protobuf before sent over wire

### PR DESCRIPTION
**Pull request by bergquist**
I forgot to map it to protobuf. My bad

### CODE REVIEW

File: pkg/plugins/backendplugin/backend_plugin.go [hunk 1]
```diff
@@ -205,6 +205,27 @@ func (p *BackendPlugin) checkHealth(ctx context.Context, config *PluginConfig) (
 		LastUpdatedMS:           config.Updated.UnixNano() / int64(time.Millisecond),
 	}
 
+	if config.DataSourceConfig != nil {
+
```
Comment (bergquist) [inline pkg/plugins/backendplugin/backend_plugin.go]:
```suggestion
```

**Review by aknuds1 — APPROVED** [review]
LGTM

**Review by marefr — APPROVED** [review]
LGTM

**Comment by aknuds1** [comment]
Try merging with latest master, in order to fix build.

**Review by ryantxu — APPROVED** [review]
That would do it!

**Comment by bergquist** [comment]
in favor of https://github.com/grafana/grafana/pull/22848

**Pull request closed without merging**